### PR TITLE
[50] Implement sign up page for actor

### PR DESCRIPTION
**Pull request by miello**
## What did you do

- Implement actor signup page 
- Send form to backend when submit

## Demo

- https://dev.modela.miello.dev/signup/actor

image

### CODE REVIEW

File: apps/web/src/modules/signup/actor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        <Grid container spacing={2} sx={{ padding: '12px' }}>
+          <Grid item xs={12} sm={12}>
+            <Typography variant="h5" sx={{ textAlign: 'center' }}>
+              นักแสดง
+            </Typography>
+          </Grid>
+          <Grid item xs={12} sm={12}>
+            <Divider />
+          </Grid>
+          <Grid item xs={12} sm={6}>
```
Comment (PKhing) [inline apps/web/src/modules/signup/actor/index.tsx:39]: (Optional) ยัดใส่ constant แล้ว map เอามั้ย ยาวชิบ 55555

File: apps/web/src/modules/signup/components/UploadFile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            variant="subtitle2"
+            fontWeight={600}
+            color="primary"
+          >
+            {name}
+          </Link>
+        </Typography>
+      ) : (
+        <></>
+      )}
```
Comment (PKhing) [inline apps/web/src/modules/signup/components/UploadFile/index.tsx:51]:
```suggestion
      {name && !error && (
        <Typography sx={{ textAlign: 'center' }} variant="subtitle2">
          <Link
            href={url}
            target="_blank"
            rel="noopener"
            variant="subtitle2"
            fontWeight={600}
            color="primary"
          >
            {name}
          </Link>
        </Typography>
      )}
```

**Review by PKhing — CHANGES_REQUESTED** [review]

**Comment by PKhing** [comment]
reduce padding and increase navbar z index 
image

**Comment by PKhing** [comment]
should we guard this page so that the user can access only when not logged in

**Comment by miello** [comment]
> should we guard this page so that the user can access only when not logged in

It should but currently I have hard-coded guard process into actor signup page for now. Page guard for login, signup will be refactored when I do the casting signup page

File: apps/web/src/modules/signup/actor/index.tsx (hunk 1 above)
Comment (miello) [reply apps/web/src/modules/signup/actor/index.tsx:54]: ขอทำรวมกับ Casting Signup ได้ไหมครับ 😂

**Review by PKhing — APPROVED** [review]

**Pull request merged**